bin/check-haproxy.rb

```diff
@@ -126,19 +134,39 @@ def run #rubocop:disable all
       critical_sessions = services.select { |svc| svc[:slim].to_i > 0 && (100 * svc[:scur].to_f / svc[:slim].to_f) > config[:session_crit_percent] }
       warning_sessions = services.select { |svc| svc[:slim].to_i > 0 && (100 * svc[:scur].to_f / svc[:slim].to_f) > config[:session_warn_percent] }
 
+      critical_backends = services.select do |svc|
+        config[:backend_session_crit_percent] &&
+        svc[:svname] == 'BACKEND' &&
+        svc[:smax].to_i > 0 &&
+        (100 * svc[:scur].to_f / svc[:smax].to_f) > config[:backend_session_crit_percent]
```

Comment:
If default isn't set, config[:backend_session_crit_percent] is nil and therefore it returns and empty array every time